theseus/utils/examples/__init__.py

```diff
@@ -3,26 +3,47 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-from .motion_planning import (
-    InitialTrajectoryModel,
-    MotionPlanner,
-    ScalarCollisionWeightAndCostEpstModel,
-    ScalarCollisionWeightModel,
-    TrajectoryDataset,
-    generate_trajectory_figs,
-)
-from .tactile_pose_estimation import (
-    TactileMeasModel,
-    TactilePoseEstimator,
-    TactilePushingDataset,
-    TactileWeightModel,
-    create_tactile_models,
-    get_tactile_cost_weight_inputs,
-    get_tactile_initial_optim_vars,
-    get_tactile_motion_capture_inputs,
-    get_tactile_nn_measurements_inputs,
-    get_tactile_poses_from_values,
-    init_tactile_model_from_file,
-    update_tactile_pushing_inputs,
-    visualize_tactile_push2d,
-)
+try:
+    from .motion_planning import (
+        InitialTrajectoryModel,
+        MotionPlanner,
+        ScalarCollisionWeightAndCostEpstModel,
+        ScalarCollisionWeightModel,
+        TrajectoryDataset,
+        generate_trajectory_figs,
+    )
+except ModuleNotFoundError:
+    print(
```

Comment:
This print was replaced to warning in #119. Might make sense to rebase on main after #119 is merged.